app/src/main/java/net/gsantner/markor/format/markdown/MarkdownHighlighterPattern.java

```diff
@@ -17,7 +17,7 @@
     HEADING(Pattern.compile("(?m)((^#{1,6}[^\\S\\n][^\\n]+)|((\\n|^)[^\\s]+.*?\\n(-{2,}|={2,})[^\\S\\n]*$))")),
     HEADING_SIMPLE(Pattern.compile("(?m)(#{1,6}\\s.*$)")),
     LINK(Pattern.compile("\\[([^\\[]+)\\]\\(([^\\)]+)\\)")),
-    LIST_UNORDERED(Pattern.compile("(\\n|^)\\s{0,12}([*+-])( \\[[ xX]\\])?(?= )")),
+    LIST_UNORDERED(Pattern.compile("(\\n|^)\\s{1,16}([*+-])( \\[[ xX]\\])?(?= )")),
```

Comment:
I made this change to handle 4 indent levels. I can revert this change if it is likely to cause issues elsewhere

Reply:
tbh, if you make 4 indent levels in markdown you make something wrong :D. Just whitespace a whole line then a list character and horizontal space is over :D

Reply:
True :). However I write a lot on the desktop and make small changes on the go in Markor. On the desktop I use lots of indentation and so I would like to support it in Markor as well. 

Is there a reason we can't use \\s*?